src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1564,10 +1560,6 @@
     text: |
         az storage share url --account-key 00000000 --account-name MyAccount --name MyFileShare
     crafted: true
-  - name: Create a URI to access a file share. (autogenerated)
```

Comment:
The delete is not expected, either.
As we discussed before, we would like a different examples name rather than removing the example with the same name.

Reply:
Aladdin may found the usage of this one is dropping down dramatically, so removed it and not recommended.

Reply:
The example is a correct sample and it should be fine to keep it. But I am not sure if it is deleted because of the same example name with a previous one. We can confirm the reason with Aladdin team.

Reply:
Hi, Roshanak, @albaloo, could you help to answer this?

Reply:
> 1. it's deleted because the other one is more frequent not because of the title
> 2. If you think a particular example is good for some reason and it shouldn't be deleted, let us know and we see if we can identify a pattern to take care of it
> 3. To elaborate more on my answer above, on each run we look at the usage in past 90 days to find out popular trends and we sort and update examples based on that